terraform/shared/shared.tf.json.template.py

```diff
@@ -326,14 +326,37 @@ def conformance_pack(name: str) -> str:
                 }
             },
             'logs': {
+                'bucket': '${aws_s3_bucket.logs.id}',
```

Comment:
The logs bucket in `prod` already has a lifecycle config for some reason. This will conflict with that. Please investigate.

Reply:
`expire` is the current `logs` bucket lifecycle rule, which is being updated by this PR from 90 days to 365.
https://github.com/DataBiosphere/azul/blob/504fd8c9bb01ef071d7231036e35ea74a20e8ca7/terraform/shared/shared.tf.json.template.py#L337
Originally in this PR I had split this lifecycle rule in to three, one per top-level subdirectory in the bucket, however I have reverted that change based on your review comment below.